irc/test.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+// runs all the tests in this module
+import "./mod_test.ts";
```

Comment:
We haven't fully defined these conventions .. but I prefer either `test.ts` or `mod_test.ts` but not both. They both suggest it's the file that could be run to test all the entire module.